buildSrc/win.gradle

```diff
@@ -74,46 +85,33 @@ setupTools("windows_tools",
+        defineProperty("WINDOWS_VS_VSINSTALLDIR", properties, "")
+        defineProperty("WINDOWS_VS_WINSDKDLLINSTALLDIR", properties, "")
+        defineProperty("WINDOWS_SDK_DIR", properties, "")
         defineProperty("WINDOWS_SDK_VERSION", properties, "")
         defineProperty("WINDOWS_VS_VCINSTALLDIR", properties, "$WINDOWS_VS_VSINSTALLDIR/VC")
         defineProperty("WINDOWS_VS_DEVENVDIR", properties, "$WINDOWS_VS_VSINSTALLDIR/Common7/IDE")
-        defineProperty("WINDOWS_VS_DEVENVCMD", properties, "$WINDOWS_VS_DEVENVDIR/VCExpress.exe")
         defineProperty("WINDOWS_VS_MSVCDIR", properties, WINDOWS_VS_VCINSTALLDIR)
         defineProperty("WINDOWS_VS_VC_TOOLS_INSTALL_DIR", properties, "")
         defineProperty("WINDOWS_VS_VC_TOOLS_REDIST_DIR", properties, "")
-        defineProperty("WINDOWS_DXSDK_DIR", properties, System.getenv().get("DXSDK_DIR"))
-        defineProperty("WINDOWS_VS_INCLUDE", properties, "$WINDOWS_VS_VCINSTALLDIR/INCLUDE;" + "$WINDOWS_SDK_DIR/include;")
-        defineProperty("WINDOWS_VS_VER", properties, "150")
-        defineProperty("WINDOWS_CRT_VER", properties, "150")
-        defineProperty("WINDOWS_VS_LIB", properties, "$WINDOWS_VS_VCINSTALLDIR/LIB;" + "$WINDOWS_SDK_DIR/lib;")
-        defineProperty("WINDOWS_VS_LIBPATH", properties, "$WINDOWS_VS_VCINSTALLDIR/LIB;")
-        def parfaitPath = IS_COMPILE_PARFAIT ? System.getenv().get("PARFAIT_PATH") + ";" : "";
-        defineProperty("WINDOWS_VS_PATH", properties, parfaitPath + "$WINDOWS_VS_DEVENVDIR;" +
-                "$WINDOWS_VS_VSINSTALLDIR/VC/BIN;" +
-                "$WINDOWS_VS_VSINSTALLDIR/Common7/Tools;" +
-                "$WINDOWS_VS_VCINSTALLDIR/VCPackages;" +
-                "$WINDOWS_SDK_DIR/bin/NETFX 4.0 Tools;" +
-                "$WINDOWS_SDK_DIR/bin;" +
-                System.getenv().get("PATH"))
+        defineProperty("WINDOWS_VS_INCLUDE", properties, "")
+        defineProperty("WINDOWS_CRT_VER", properties, "143")
+        defineProperty("WINDOWS_VS_LIB", properties, "")
+        defineProperty("WINDOWS_VS_LIBPATH", properties, "")
+        defineProperty("WINDOWS_VS_PATH", properties, "")
     }
 )
 
 if (WINDOWS_SDK_DIR == null || WINDOWS_SDK_DIR == "") {
-    throw new GradleException("FAIL: WINSDK_DIR not defined");
+    throw new GradleException("Cannot locate Visual Studio compilers; set the 'VSCOMNTOOLS' env var\nto the 'VC/Auxiliary/Build' dir in your Visual Studio installation");
 }
 
 // Define set of flags shared for all targets that support debug compilation
 def ccDebugFlags =
     IS_DEBUG_NATIVE ? ["/MDd", "/Od", "/Zi", "/DDEBUG"] : ["/O2", "/MD", "/DNDEBUG"]
 
-def winVsVer = Integer.parseInt(WINDOWS_VS_VER)
-if (winVsVer >= 120) {
-    // Serialize access to PDB file for debug builds if on VS2013 or later
-    if (IS_DEBUG_NATIVE) ccDebugFlags += "/FS"
-}
+// Serialize access to PDB file for debug builds if on VS2013 or later
```

Comment:
The "if on VS2013 or later" part of the comment is obsolete.

Reply:
I'll remove it.